src/components/screens/ScoreSetView.vue

```diff
@@ -461,6 +462,15 @@ export default {
       const showCountColumns = !_.isEmpty(this.item?.datasetColumns?.countColumns)
       return showCountColumns ? [...fixedColumns, ...this.item?.datasetColumns?.countColumns || []] : []
     },
+    showHeatmap() {
+      // Threshold is 5%.
+      const sparsityThreshold = 0.05
```

Comment:
I think this should be a global variable `SPARSITY_THRESHOLD`, defined within `ScoreSetHeatmap.ts`.

Reply:
I don't find either `SPARSITY_THRESHOLD` or `ScoreSetHeatmap.ts`. Search results shows nothing about `SPARSITY_THRESHOLD` and that's why I wrote a local threshold.

Reply:
Sorry I wasn't clear about this. It doesn't exist at the moment, I just think it should be created as a global variable rather than a local one.

Reply:
Got it. It makes sense now. I'll modify it.